site-src/v1alpha2/references/cross-namespace-references.md

```diff
@@ -0,0 +1,273 @@
+# Cross namespace references and ReferencePolicy
+
+In the Gateway API, it is possible to have references between objects cross namespace boundaries.
+In particular, it's expected that Gateways and Routes may exist in different namespaces,
+or that Services may be referred to by Routes in a another namespace.
+
+However, this significantly violates the idea of a namespace as the edge of a trust domain.
+In order to bring cross-namespace references under the control of the owner of the referent object's namespace,
+the Gateway API has a ReferencePolicy object that must be created in the referent namespace for the reference to be successful.
```

Comment:
nit: 80 char limit throughout the file

Reply:
> object that must be created in the referent namespace

I know we use 'referent' a lot and assume that it is understood by our users but I think explaining the referent and referee would help here. This paragraph is very hard to parse for me even though I'm familiar with the concept.

Reply:
Added a note on terminology at the start, I think that's a great idea.